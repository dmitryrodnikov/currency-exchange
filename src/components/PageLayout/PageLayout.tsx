import * as React from 'react';
import styled from 'styled-components';

const StyledPageLayout = styled.div`
    padding: 40px;
    max-width: 600px;
    margin: 0 auto;
`;

export const PageLayout: React.FunctionComponent = ({children}) => {
    return (
        <StyledPageLayout>{children}</StyledPageLayout>
    );
};
