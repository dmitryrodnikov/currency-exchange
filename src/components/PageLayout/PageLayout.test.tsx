import React from 'react';
import {shallow, render} from 'enzyme';
import {PageLayout} from './PageLayout';

const PAGE_CONTENT = 'Test';

describe('PageLayout', () => {
    it('Shallow render snapshots are equal', () => {
        const wrapper = shallow(<PageLayout>{PAGE_CONTENT}</PageLayout>);
        expect(wrapper).toMatchSnapshot();
    });

    it('Render snapshots are equal', () => {
        const wrapper = render(<PageLayout>{PAGE_CONTENT}</PageLayout>);
        expect(wrapper).toMatchSnapshot();
    });
});
