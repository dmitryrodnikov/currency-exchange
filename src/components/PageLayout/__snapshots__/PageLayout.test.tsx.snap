// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout Render snapshots are equal 1`] = `
<div
  class="sc-bdVaJa cGCNPj"
>
  Test
</div>
`;

exports[`PageLayout Shallow render snapshots are equal 1`] = `
<styled.div>
  Test
</styled.div>
`;
